packages/forma-36-website/gatsby-config.js

```diff
@@ -3,9 +3,11 @@ const path = require('path');
 module.exports = {
   siteMetadata: {
     title: 'Forma 36 - The Contentful Design System',
-    promoText: '',
-    promoLink: '',
-    promoLinkText: '',
+    promoText: 'Introducing a visual refresh to Forma 36 components',
```

Comment:
Should we call them React components like we do elsewhere?

Reply:
hmm, good point. On the website, the audience is not only engineers, so maybe it's not so important information, but sure, for consistency reasons I can adjust it.

Reply:
@denkristoffer ✅